drf_hal_json/serializers.py

```diff
@@ -30,16 +31,12 @@ def data(self):
         )
 
 
-class HalModelSerializer(HyperlinkedModelSerializer):
-    """
-    Serializer for HAL representation of django models
-    """
-    serializer_related_field = HyperlinkedRelatedField
-    serializer_url_field = HalHyperlinkedIdentityField
+class HalSerializerMixin(Serializer):
```

Comment:
```suggestion
class HalSerializer(Serializer):
```

I guess we can call it `HalSerializer` since if it inherits, it's not really a mixin.